.github/workflows/stale.yaml

```diff
@@ -20,7 +20,8 @@ jobs:
           # Pull Requests become stale more quickly due to merge conflicts.
           # Also, we promote minimizing WIP.
           days-before-pr-stale: 7
-          days-before-pr-close: 3
+          days-before-issue-stale: 30
```

Comment:
Default value is 60 which feels too long if we want to keep issue count small. Is 30 too aggressive?

Reply:
I think 60 (or even 90) are better knobs based on my past experiences with stale bots. I worry that at 30 the stale bot spam will be more annoying than the issue count.

Based on pulse, the issue count has been shrinking naturally, so I don't think we should make this change.
<img width="930" alt="Screen Shot 2022-08-24 at 12 49 31 PM" src="https://user-images.githubusercontent.com/7416144/186488403-3790bd62-ade1-463e-9e75-5f41d6a08a9a.png">

Reply:
Good data to back that up. I will remove